apps/cli/package.json

```
@@ -1,11 +1,12 @@
 {
-  "name": "xstate-cli",
+  "name": "@xstate/cli",
   "version": "0.0.1",
   "bin": {
     "xstate": "./bin/bin.js"
```

WDYT about making this `xstate-cli` to differentiate from the core?

---

Oh, I'm on board with `@xstate/cli` for the package name. I was asking about the name of this executable:
```suggestion
    "xstate-cli": "./bin/bin.js"
```

---

Ah, gotcha. @davidkpiano?

I like `xstate` - short and snappy.

---

Just two examples from the wild:

webpack is using webpack-cli
vue is using vue

---

@mattpocock `xstate` is fine 👍